src/components/proposals/brand/navbar/navbar.scss

```diff
@@ -0,0 +1,466 @@
+
+
+  .site-links.responsive{
+    padding:10px;
+    background-color: white;
+  }
+
+  .site-links.responsive a,
+  .site-links.responsive .dropdown {
+    float: none;
+    display: block;
+
+  }
+
+  .site-links.responsive {
+    position: absolute;
+    width: 100%;
+    box-shadow: 0 5px 10px -2px rgba(0, 0, 0, 0.26);
+    -webkit-box-shadow: 0 5px 10px -2px rgba(0, 0, 0, 0.26);
+  }
+
+  .site-links.responsive a {
+    color: $secondary;
+    text-decoration: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 10px;
+    align-items: center;
+   // margin: 0 auto;
+    text-align: center;
+    height: 50px;
+    width:100%;
+    font-size: 1.1375rem;
+
+    background-color: white;
+    border-bottom: 1px solid rgba(0,0,0,0.2);
+    border-top: 1px solid rgba(255,255,255,0.05);
+  }
+
+  .dropdown .dropdown-content{
+
+    a{
+      color: #666;
+      font-weight: 400;
+      font-size: 1.2rem;
+    }
+
+    a:hover {
+
+      text-decoration: underline ;
+    }
+
+  }
+
+
+
+  .site-links a:hover {
+    background-color: $white;
+    color: $secondary;
+
+    span {
+      border-bottom-color: $secondary;
+    }
+  }
+
+  .responsive .open {
+    position: relative;
+    display: block;
+    width: 100%;
+    border-top: none;
+    z-index: 1;
+    box-shadow: none;
+    -webkit-box-shadow: none;
+    top:0px;
+  }
+
+}
+
+//desktop
+@media screen and (min-width: 855px) {
+
+  .links{
+    padding-top: 10px;
+  }
+
+  .uids .site-links a {
+    float: left;
+    display: block;
+  }
+
+  .wrap-links{
+    display: flex;
+    flex-wrap:wrap;
+    flex: 1;
+  }
+
+  .site-links a {
+    white-space: nowrap;
+  }
+
+  .top-nav {
+    display: flex;
+    flex: 1;
+    flex-direction: row;
+    justify-content:flex-start;
+  }
+
+    .site-name-wrapper{
+    display: flex;
+    flex: 1;
+    width:100%;
+    padding-left:11px;
+  }
+
+  .toggle {
+    display: none;
+  }
+
+  .top{
+    min-height: 50px;
+    // display: flex;
+    // flex-direction: row;
+    // justify-content: space-between;
+  }
+
+  .site-links{
+    min-height: 50px;
+    position: relative;
+    display: flex;
+
+    flex-direction: row;
+    justify-content:flex-end;
+    background-color: $primary;
+
+  }
+
+
+  .links{
+    display: flex;
+    width: auto;
+    flex-direction: row;
+  }
+
+  .align-links{
+   //padding-top: 6px;
+
+  }
+
+  .site-links {
+
+    display: flex;
+    flex:1;
+
+
+    a {
+      padding: 0px 10px 0px 10px;
+      display: flex;
+      float: left;
+      justify-content: center;
+      align-items: center;
+      margin: 0 auto;
+
+      color: $secondary;
+      text-align: center;
+      text-decoration: none;
+      font-size: 17px;
+      height: 100%;
+      min-width: 90px;
+      border-bottom: 3px solid;
+      border-bottom-color:$primary;
+    }
+
+    a >span>i:last-child{
+      margin-left:5px;
+    }
+
+    a>span>i:first-child{
+      margin-right:5px;
+    }
+
```

Comment:
Add a rule to adjust icon size and color?  Something smaller and a different color like: 
```    
color: #fff;
font-size: .7rem;
line-height: 1.4;
```